pepper-apis/src/main/resources/org/broadinstitute/ddp/service/KitCheckService.sql.stg

```diff
@@ -27,4 +27,6 @@ select user.user_id,
           and kit.study_id = study.umbrella_study_id
        )
    and valid_to is null
+ order by user.user_id
+ limit :limit offset :offset
```

Comment:
Since you will be consuming the users in incrementally instead of one big transaction, it could happen that your overall result set (i.e., the set of all rows) you are trying to consume could change while you are iterating through them.
Consider using user_study_enrollment.valid_from as the ORDER BY key . Presumably that will put new enrollees at the end of the result set. `user_id` might or might not.

Reply:
I liked ordering by `user_id` since that's deterministic. But I see your point about new enrollments. I had assumed that new users coming into study will be new user (i.e. new user ids). Perhaps there might be a chance that old users (hence, old user id) can come in to. I guess it will be safer to BOTH `order by valid_from asc, user_id asc`.

Reply:
Just saw your other comment in other PR. I like `user_study_enrollment_id`, I'll use that instead.